src/main/index.ts

```diff
@@ -61,11 +61,15 @@ import { FilesystemProvisionerStore } from "./extension-filesystem";
 import { SentryInit } from "../common/sentry";
 import { ensureDir } from "fs-extra";
 import { Router } from "./router";
-import { initMenu } from "./menu";
+import { initMenu } from "./menu/menu";
 import { initTray } from "./tray";
 import { kubeApiRequest, shellApiRequest, ShellRequestAuthenticator } from "./proxy-functions";
 import { AppPaths } from "../common/app-paths";
 import { ShellSession } from "./shell-session/shell-session";
+import { getDi } from "./getDi";
+import electronMenuItemsInjectable from "./menu/electron-menu-items.injectable";
```

Comment:
why is this "electron" menu items?

Reply:
Since `menuItems` is not enough to make it obvious. Other option was `appMenuItems` but that has same effect. App can be anything. Using Electron in name is not best, but still lot better than without it. Maybe `mainAppMenuItems`?